modules/core/shared/src/main/scala/gem/enum/GsaoiFilter.scala

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import gem.math.Wavelength
+import gem.util.Enumerated
+import scala.concurrent.duration.FiniteDuration
+
+/**
+ * Enumerated type for GSAOI Filter.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GsaoiFilter(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val wavelength: Wavelength,
+  val readMode: GsaoiReadMode,
+  val exposureTime5050: FiniteDuration,
+  val exposureTimeHalfWell: FiniteDuration,
+  val band: Option[MagnitudeBand]
+) extends Product with Serializable
+
+object GsaoiFilter {
+
+  /** @group Constructors */ case object Z extends GsaoiFilter("Z", "Z", "Z (1.015 um)", Wavelength.fromPicometers.unsafeGet(1020000), GsaoiReadMode.Faint, new FiniteDuration(26, java.util.concurrent.TimeUnit.SECONDS), new FiniteDuration(4619, java.util.concurrent.TimeUnit.SECONDS), Some(MagnitudeBand.J))
+  /** @group Constructors */ case object HeI extends GsaoiFilter("HeI", "HeI", "HeI (1.083 um)", Wavelength.fromPicometers.unsafeGet(1080000), GsaoiReadMode.VeryFaint, new FiniteDuration(72, java.util.concurrent.TimeUnit.SECONDS), new FiniteDuration(21792, java.util.concurrent.TimeUnit.SECONDS), Some(MagnitudeBand.J))
```

Comment:
In the old codebase, the `exposureTime5050` is `72.6` seconds.  Did this change or should it be 72600 milliseconds here?  The same question applies for the other subsecond resolution times.

Reply:
Good question. I need to revisit why the fraction is being lost. Am I right that on the db we want to store `72.6` rather than `72600`?

Reply:
Right, I think ideally everything will appear in units that the astronomer is expecting.